tests/Traits/ComposerTestTrait.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Pantheon\Terminus\Tests\Traits;
+
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Trait ComposerTestTrait.
+ *
+ * @package Pantheon\Terminus\Tests\Traits
+ */
+trait ComposerTestTrait
+{
+    /**
+     * Run a composer command.
+     *
+     * @param string $command
+     *   The command to run.
+     * @param string|null $pipeInput
+     *   The pipe input.
+     *
+     * @return array
+     *   The execution's stdout [0], exit code [1] and stderr [2].
+     */
+    protected static function callComposer(string $command, ?string $pipeInput = null): array
+    {
+        $procCommand = sprintf('%s %s', getenv('COMPOSER_BIN_FILE'), $command);
```

Comment:
Using just 'composer' as a default for this var would work? Or does it need the full path?

Reply:
Yep, definitely. My question is: is it possible to run this with a safe default so that you may not set the env var under some "common" circumstances?

Reply:
I guess we could add it to the config?